pkg/sync/v4/translate.go

```diff
@@ -544,6 +536,18 @@ var translations = map[string][]struct {
 			Template: "{{ String (PrivateKeyAsBytes .Config.Certificates.Router.Key) }}",
 		},
 	},
+	"Secret/default/docker-registry-http": {
+		{
+			Path:     jsonpath.MustCompile("$.stringData.password"),
+			Template: "{{ .Config.RegistryHTTPSecret }}",
```

Comment:
Missing Base64Encode here I think

Reply:
No, this is going to `stringData` not `data`